From e635cfeed399da87d8142dc19017ad686cd90fa5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Mon, 18 Feb 2019 17:10:54 +0100
Subject: [PATCH] Linux: disable AMD for sure

For some reason this was failing builds. This is just hacky fix and
should be dropped when upstream fixes list of disabled options.
---
 target/linux/generic/config-4.14 | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/target/linux/generic/config-4.14 b/target/linux/generic/config-4.14
index 8fd0d469fd..b59d2c59d3 100644
--- a/target/linux/generic/config-4.14
+++ b/target/linux/generic/config-4.14
@@ -1199,7 +1199,8 @@ CONFIG_DQL=y
 # CONFIG_DRM_PARADE_PS8622 is not set
 # CONFIG_DRM_PL111 is not set
 # CONFIG_DRM_QXL is not set
-# CONFIG_DRM_RADEON is not set
+CONFIG_DRM_RADEON=n
+CONFIG_DRM_AMDGPU=n
 # CONFIG_DRM_RCAR_DW_HDMI is not set
 # CONFIG_DRM_SII902X is not set
 # CONFIG_DRM_SIL_SII8620 is not set
-- 
2.20.1

